source/_patterns/01-molecules/components/comment.mustache

```diff
@@ -0,0 +1,13 @@
+<div class="comment">
+  <div class="comment__avatar">
+    <img alt="" src="{{{ comment_avatar }}}" class="avatar" height="32" width="32">
+  </div>
+  <div class="comment__body">
+    <div class="comment__meta">
+      <span class="byline font--secondary--s gray can-be--white">{{{ comment_byline }}}</span>
+      <span class="divider">|</span>
+      <span class="pub_date font--secondary--s gray can-be--white">{{{ comment_date }}}</span>
+    </div>
+    <p class="comment__text">{{{ comment_text }}}</p>
+  </div>
+</div>
```

Comment:
@kelscahill There's a few other nuances to these comments. Like, comment threads that involve nested comments and validation states. For an example of all of the possible comment variations, view the WordPress theme on another theme with comments turned on. 

Also, can you please add the comments organism to the News Article page so we can see it in action?